src/loaders/bitmapFontParser.js

```diff
@@ -67,29 +67,54 @@ export default function ()
             xmlUrl += '/';
         }
 
-        const textureUrl = xmlUrl + resource.data.getElementsByTagName('page')[0].getAttribute('file');
+        const pages = resource.data.getElementsByTagName('page');
+        let textureUrl;
+        const textures = [];
+        const loadOptions = {
+            crossOrigin: resource.crossOrigin,
+            loadType: Resource.LOAD_TYPE.IMAGE,
+            metadata: resource.metadata.imageMetadata,
+            parentResource: resource,
+        };
 
-        if (utils.TextureCache[textureUrl])
+        for (let x = 0; x < pages.length; ++x)
         {
-            // reuse existing texture
-            parse(resource, utils.TextureCache[textureUrl]);
-            next();
-        }
-        else
-        {
-            const loadOptions = {
-                crossOrigin: resource.crossOrigin,
-                loadType: Resource.LOAD_TYPE.IMAGE,
-                metadata: resource.metadata.imageMetadata,
-                parentResource: resource,
-            };
+            textureUrl = xmlUrl + pages[x].getAttribute('file');
 
-            // load the texture for the font
-            this.add(`${resource.name}_image`, textureUrl, loadOptions, (res) =>
+            if (utils.TextureCache[textureUrl])
             {
-                parse(resource, res.texture);
-                next();
-            });
+                textures.push(utils.TextureCache[textureUrl]);
+            }
+            else
+            {
+                // load the texture for the font
+                this.add(`${resource.name}_image${x}`, textureUrl, loadOptions, () =>
+                {
+                    const textures = [];
+                    let textureUrl;
```

Comment:
I would suggest using different variable name.

Re-defining upperscope variables can lead to mistakes.

On line 98:

```
const nextTextureUrl = ...;
```